PocketKit/Sources/Sync/CoreData+Relationships/README.md

```diff
@@ -1,6 +1,6 @@
 #  CoreData Relationships
 
-At Pocket we use NSFetchedResultsControllers to power our data views. These work great and update our views only when the object they are observing updates it's values. However it has a downside where CoreData will not tell NSFetchedResultsController to update if the object it is montioring has an update to any of its related objects.
+At Pocket we use NSFetchedResultsControllers to power our data views. These work great and update our views only when the object they are observing updates its values. However it has a downside where Core Data will not tell NSFetchedResultsController to update if the object it is montioring has an update to any of its related objects.
```

Comment:
maybe we can reference the terminology `RichFetchRequest`

Reply:
maybe, but since refactoring this piece of documentation is out of scope of this PR, we can do it in a separate one. The typos catched my eyes and I could not resist fixing them 😄